2019/2019_File_Generation_Features.md

```diff
@@ -0,0 +1,67 @@
+# 2019 File Generation Features
```

Comment:
change this filename to readme.md so that it renders in the sub folder.

Reply:
Please make appropriate edits based on the comments to the 2018 version of the file.